capa/ghidra/capa_explorer.py

```diff
@@ -359,9 +363,26 @@ def main():
         popup("capa explorer found no matches.")  # type: ignore [name-defined] # noqa: F821
         return capa.main.E_EMPTY_REPORT
 
-    for item in parse_json(capa_data):
-        item.bookmark_functions()
-        item.label_matches()
+    user_choice = askChoice(  # type: ignore [name-defined] # noqa: F821
+        "Choose b/w bookmarks & comments",
+        "preferred action:",
+        ["bookmarks", "comments", "both", "none"],
```

Comment:
```suggestion
        ["Bookmarks", "Comments", "All"],
```

Reply:
I don't see a use case for `None` unless you see one?

Reply:
@mike-hunhoff    
yeah, there is no use case for none.
Also actually, now do we need "All?" that we are going to use askChoices; we can just let the user choose between the three, it's gonna o/p a list anyway, so we just iterate through it and mark some boolean values accordingly like something like this and make changes to the functions to accommodate the choice 
![image](https://github.com/mandiant/capa/assets/48616823/abcd24e8-74cc-4ebc-bd82-720d45f2edc7)

Reply:
I believe Ghidra's FlatProgramAPI method `askChoices` automatically provides an "All" option. I'm not sure what is returned in `user_choices` when that option is selected but it appears that the code you've shown above _should_ handle it.